runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -611,6 +611,19 @@ J9::AheadOfTimeCompile::initializeCommonAOTRelocationHeader(TR::IteratedExternal
          }
          break;
 
+      case TR_ValidateDefiningClassFromCP:
+         {
+         TR_RelocationRecordValidateDefiningClassFromCP *dcpRecord = reinterpret_cast<TR_RelocationRecordValidateDefiningClassFromCP *>(reloRecord);
+
+         TR::DefiningClassFromCPRecord *svmRecord = reinterpret_cast<TR::DefiningClassFromCPRecord *>(relocation->getTargetAddress());
+
+         dcpRecord->setiSStatic(reloTarget, svmRecord->_isStatic);
```

Comment:
should this be `setIsStatic` ?

Reply:
Heh yeah it should; good catch. Fixed in https://github.com/eclipse/openj9/pull/10298/commits/10a0d9da2b8910f8be1e86aed05ae3bad0d7211c.